src/deployment/deploy.py

```diff
@@ -397,8 +406,15 @@ def try_sp_create() -> None:
         ):
             set_app_audience(app.object_id, "AzureADMyOrg")
             url = "https://%s.azurewebsites.net" % self.application_name
+            api = "api://%s.azurewebsites.net" % self.application_name
+
+            if not app.identifier_uris.contains(api_url):
+                identifier_uris = [url, api_url]
+            else:
+                identifier_uris = [api_url]
```

Comment:
Actually it might better to just add the new url instead of overriding in case the application registration was manually updated
```suggestion 
            identifier_uris = app.identifier_uris
            if not identifier_uris.contains(api_url):
                identifier_uris.append(api_url)
```

Reply:
then we'll be appending every time the code runs, so we might end up with 10 of those

Reply:
Every time we add to existing and then patch with new array, so we'll accumulate api_urls